web_notify/__init__.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=missing-docstring
```

Comment:
Coule you remove the useless pylint disable in all the files ?

Reply:
removed..